pkgs/development/libraries/qt-6/srcs.nix

```diff
@@ -115,6 +115,14 @@
       name = "qtmultimedia-everywhere-src-6.4.3.tar.xz";
     };
   };
+  qtmqtt = {
```

Comment:
`# DO NOT EDIT! This file is generated automatically.`

Reply:
Ups... I missed that.

I tweaked it but now there is a version specified that is not part of that srcs and I feel that it is going to be way too easy to introduce version mismatch in Qt this way.

Reply:
Shouldn't the build fail on version mismatch? That way we can easily catch it with nixpkgs-review.